wmfdata/hive.py

```diff
@@ -31,30 +51,12 @@ def run_cli(
         * "raw": a TSV string, as returned by the command line interface.
     * `heap_size`: the amount of memory available to the Hive client. Increase
       this if a command experiences an out of memory error.
-    * `use_nice`: Run with a lower priority for processor usage.
-    * `use_ionice`: Run with a lower priority for disk access.
     """
 
-    commands = ensure_list(commands)
     if format not in ["pandas", "raw"]:
-        raise ValueError("'{}' is not a valid format.".format(format))
-    if format == "raw":
```

Comment:
Could you move this deprecation warning to the `run` function?

Reply:
but "raw" isn't deprecated, it still works, no?

Reply:
Well, I take "deprecated" to mean that it's recommended not to be used. And the existing code has a recommendation not to use the raw format (as with the other run functions). So I want to make sure that continues to be the case.

If you think we should _keep_ the raw format, that's a valid conversation, but a separate one 😊

Reply:
I see.  I am not a user so I don't know, I'd imagine raw would be useful if you didn't want to get pandas involved, but maybe that's just always better.  I noticed somehow `pd.read_sql` from below seems faster than the raw fetch.  If it's really going to be removed in the next major release, then totally, let's deprecate (done)

Reply:
I see what you mean. I'm going to resolve this comment and move the discussion to https://phabricator.wikimedia.org/T300442, to keep this conversation focused on changing the Hive backend.